<?xml version="1.0" encoding="utf-8"?>
<TestRun id="3cd0eb63-7da2-4cd5-8585-65bbb518ac37" name="@hamilton 2024-03-28 13:56:40" xmlns="http://microsoft.com/schemas/VisualStudio/TeamTest/2010">
  <Times creation="2024-03-28T13:56:40.2711995+00:00" queuing="2024-03-28T13:56:40.2711996+00:00" start="2024-03-28T13:56:35.6880886+00:00" finish="2024-03-28T13:56:40.3043045+00:00" />
  <TestSettings name="default" id="297faf07-3fa9-4ff7-a209-32043a706339">
    <Deployment runDeploymentRoot="_hamilton_2024-03-28_13_56_40" />
  </TestSettings>
  <Results>
    <UnitTestResult executionId="72d2656b-6d46-4313-92fb-692679f161ac" testId="9b713d3e-52f3-fa98-9430-5e9caddda841" testName="ProjectOne.Tests.UnitTest1.Test1" computerName="hamilton" duration="00:00:00.0444489" startTime="2024-03-28T13:56:39.8211031+00:00" endTime="2024-03-28T13:56:39.8211686+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="72d2656b-6d46-4313-92fb-692679f161ac">
      <Output>
        <ErrorInfo>
          <Message>This test will fail</Message>
          <StackTrace>   at ProjectOne.Tests.UnitTest1.Test1() in /home/marco/Development/emacs/trx-mode/TestData/SampleProject/ProjectOne.Tests/UnitTest1.cs:line 8
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
    <UnitTestResult executionId="3d2f0883-1f1d-442c-b8a3-81793ffadbfb" testId="4394ceb4-d8b0-a004-fc5a-c2aa357c8a4f" testName="ProjectOne.Tests.UnitTest1.Test2" computerName="hamilton" duration="00:00:00.0006674" startTime="2024-03-28T13:56:39.8313124+00:00" endTime="2024-03-28T13:56:39.8313126+00:00" testType="13cdc9d9-ddb5-4fa4-a97d-d965ccfc6d4b" outcome="Failed" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" relativeResultsDirectory="3d2f0883-1f1d-442c-b8a3-81793ffadbfb">
      <Output>
        <ErrorInfo>
          <Message>This test will fail</Message>
          <StackTrace>   at ProjectOne.Tests.UnitTest1.Test2() in /home/marco/Development/emacs/trx-mode/TestData/SampleProject/ProjectOne.Tests/UnitTest1.cs:line 14
   at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
   at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)</StackTrace>
        </ErrorInfo>
      </Output>
    </UnitTestResult>
  </Results>
  <TestDefinitions>
    <UnitTest name="ProjectOne.Tests.UnitTest1.Test1" storage="/home/marco/development/emacs/trx-mode/testdata/sampleproject/projectone.tests/bin/debug/net8.0/projectone.tests.dll" id="9b713d3e-52f3-fa98-9430-5e9caddda841">
      <Execution id="72d2656b-6d46-4313-92fb-692679f161ac" />
      <TestMethod codeBase="/home/marco/Development/emacs/trx-mode/TestData/SampleProject/ProjectOne.Tests/bin/Debug/net8.0/ProjectOne.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="ProjectOne.Tests.UnitTest1" name="Test1" />
    </UnitTest>
    <UnitTest name="ProjectOne.Tests.UnitTest1.Test2" storage="/home/marco/development/emacs/trx-mode/testdata/sampleproject/projectone.tests/bin/debug/net8.0/projectone.tests.dll" id="4394ceb4-d8b0-a004-fc5a-c2aa357c8a4f">
      <Execution id="3d2f0883-1f1d-442c-b8a3-81793ffadbfb" />
      <TestMethod codeBase="/home/marco/Development/emacs/trx-mode/TestData/SampleProject/ProjectOne.Tests/bin/Debug/net8.0/ProjectOne.Tests.dll" adapterTypeName="executor://xunit/VsTestRunner2/netcoreapp" className="ProjectOne.Tests.UnitTest1" name="Test2" />
    </UnitTest>
  </TestDefinitions>
  <TestEntries>
    <TestEntry testId="9b713d3e-52f3-fa98-9430-5e9caddda841" executionId="72d2656b-6d46-4313-92fb-692679f161ac" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestEntry testId="4394ceb4-d8b0-a004-fc5a-c2aa357c8a4f" executionId="3d2f0883-1f1d-442c-b8a3-81793ffadbfb" testListId="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
  </TestEntries>
  <TestLists>
    <TestList name="Results Not in a List" id="8c84fa94-04c1-424b-9868-57a2d4851a1d" />
    <TestList name="All Loaded Results" id="19431567-8539-422a-85d7-44ee4e166bda" />
  </TestLists>
  <ResultSummary outcome="Failed">
    <Counters total="2" executed="2" passed="0" failed="2" error="0" timeout="0" aborted="0" inconclusive="0" passedButRunAborted="0" notRunnable="0" notExecuted="0" disconnected="0" warning="0" completed="0" inProgress="0" pending="0" />
    <Output>
      <StdOut>[xUnit.net 00:00:00.00] xUnit.net VSTest Adapter v2.5.3.1+6b60a9e56a (64-bit .NET 8.0.3)
[xUnit.net 00:00:00.55]   Discovering: ProjectOne.Tests
[xUnit.net 00:00:00.87]   Discovered:  ProjectOne.Tests
[xUnit.net 00:00:00.90]   Starting:    ProjectOne.Tests
[xUnit.net 00:00:01.49]       This test will fail
[xUnit.net 00:00:01.49]       Stack Trace:
[xUnit.net 00:00:01.50]         /home/marco/Development/emacs/trx-mode/TestData/SampleProject/ProjectOne.Tests/UnitTest1.cs(8,0): at ProjectOne.Tests.UnitTest1.Test1()
[xUnit.net 00:00:01.50]            at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
[xUnit.net 00:00:01.50]            at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
[xUnit.net 00:00:01.52]       This test will fail
[xUnit.net 00:00:01.52]       Stack Trace:
[xUnit.net 00:00:01.52]         /home/marco/Development/emacs/trx-mode/TestData/SampleProject/ProjectOne.Tests/UnitTest1.cs(14,0): at ProjectOne.Tests.UnitTest1.Test2()
[xUnit.net 00:00:01.52]            at System.RuntimeMethodHandle.InvokeMethod(Object target, Void** arguments, Signature sig, Boolean isConstructor)
[xUnit.net 00:00:01.52]            at System.Reflection.MethodBaseInvoker.InvokeWithNoArgs(Object obj, BindingFlags invokeAttr)
[xUnit.net 00:00:01.52]   Finished:    ProjectOne.Tests
</StdOut>
    </Output>
    <RunInfos>
      <RunInfo computerName="hamilton" outcome="Error" timestamp="2024-03-28T13:56:39.9359312+00:00">
        <Text>[xUnit.net 00:00:01.49]     ProjectOne.Tests.UnitTest1.Test1 [FAIL]</Text>
      </RunInfo>
      <RunInfo computerName="hamilton" outcome="Error" timestamp="2024-03-28T13:56:39.9368692+00:00">
        <Text>[xUnit.net 00:00:01.52]     ProjectOne.Tests.UnitTest1.Test2 [FAIL]</Text>
      </RunInfo>
    </RunInfos>
  </ResultSummary>
</TestRun>